### PR TITLE
test(ticdc): update integration-test-binaries download

### PR DESCRIPTION
**Pull request by wk989898**
### What problem does this PR solve?

ref #10742

### What is changed and how it works?
Modify TiCDC related binaries download for integration test locally.

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: tests/integration_tests/README.md [hunk 1]
```diff
@@ -2,37 +2,9 @@
...
-   > You can also download the binaries. `sync_diff_inspector` can be downloaded 
-   > from [tidb-community-toolkit](https://download.pingcap.org/tidb-community-toolkit-v6.0.0-linux-amd64.tar.gz), 
-   > `tidb-server` related binaries can be downloaded 
-   > from [tidb-community-server](https://download.pingcap.org/tidb-community-server-v6.0.0-linux-amd64.tar.gz):
-
-   > If you are running tests on MacOS, tidb related binaries can be downloaded from tiup mirrors, such as 
-   > https://tiup-mirrors.pingcap.com/tidb-v4.0.2-darwin-amd64.tar.gz. And `sync_diff_inspector` can be compiled by 
-   > yourself from source [tidb-tools](https://github.com/pingcap/tidb-tools)
+1. Run `make prepare_test_binaries` to generate TiCDC related binaries for integration test.
+If you are running tests on MacOS Silicon, run `make prepare_test_binaries os=darwin arch=arm64`.
```
Comment (sdojjy) [inline tests/integration_tests/README.md:6]: seems we can not run integration tests on Mac

**Comment by asddongmen** [comment]
Please fill your PR description, you can reference to a merged PR in this repo.

File: tests/integration_tests/README.md [hunk 2]
```diff
@@ -2,48 +2,19 @@
...
-   > You should put `flash_cluster_manager`, `libtiflash_proxy.so` and `tiflash` into `bin` directory in TiCDC code base.
-
-   > Old versions of Minio may cause the integration test cases to fail. You can get a newer version by installing it from source
-   > ([Source Installation](https://github.com/minio/minio#install-from-source)).
-   > [RELEASE.2022-05-08T23-50-31Z](https://github.com/minio/minio/releases/tag/RELEASE.2022-05-08T23-50-31Z) is suggested.
-
-2. These are programs/packages need be installed. 
+1. Run `make prepare_test_binaries community=true` to generate TiCDC related binaries for integration test.
+You can specify version and arch, for example: `make prepare_test_binaries community=true ver=v7.0.0 arch=amd64`
+
```
Comment (CharlesCheung96) [inline tests/integration_tests/README.md:7]: Whether the description of the directory structure should be retained to facilitate troubleshooting?

**Review by CharlesCheung96 — APPROVED** [review]

**Review by sdojjy — APPROVED** [review]

**Pull request merged**